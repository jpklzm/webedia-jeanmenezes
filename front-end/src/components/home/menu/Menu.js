import React, { Component } from 'react';

import styles from './styles.scss';

class Menu extends Component {
  render() {
    return (
      <nav className="navbar navbar-default" role="navigation">
        <div className="container">
          <ul className="nav nav-justified">
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
            <li className={styles.menuItem}><a href="#" className={styles.menuText}>Lorem Ipsum</a></li>
          </ul>
        </div>
      </nav>
    );
  }
}

export default Menu;
