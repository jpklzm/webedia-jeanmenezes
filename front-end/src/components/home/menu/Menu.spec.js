import React from 'react';
import renderer from 'react-test-renderer';

import Menu from './Menu';

test('Snapshot', () => {
  const component = renderer.create(<Menu />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
