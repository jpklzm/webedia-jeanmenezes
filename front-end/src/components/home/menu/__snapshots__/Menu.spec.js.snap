// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<nav
  className="navbar navbar-default"
  role="navigation"
>
  <div
    className="container"
  >
    <ul
      className="nav nav-justified"
    >
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="/blog/1"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="#"
        >
          Lorem Ipsum
        </a>
      </li>
      <li
        className="menuItem"
      >
        <a
          className="menuText"
          href="/admin"
          onClick={[Function]}
        >
          Admin
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
